Lib9c/Action/TransferAsset.cs

```diff
@@ -84,6 +84,17 @@ public override IAccountStateDelta Execute(IActionContext context)
                 throw new InvalidTransferRecipientException(Sender, Recipient);
             }
 
+            if (state.GetState(Addresses.ActivatedAccount) is Dictionary asDict)
+            {
+                var activatedAccountsState = new ActivatedAccountsState(asDict);
+                var activatedAccounts = activatedAccountsState.Accounts;
+                if (activatedAccounts.Count != 0
+                    && !activatedAccounts.Contains(Recipient))
+                {
+                    throw new InvalidTransferRecipientException(Sender, Recipient);
```

Comment:
I am not sure. But it would be better to define the exception for this case.

Reply:
It was vague to me. `InvalidTransferRecipientExecption` is perfect fit here too. but this error also mean sender and recipient are same. so, what is your opinion? If create new exception type, what name would be better?

Reply:
Someone who catch the exception does not know the reason why the exception thrown; due to the recipient is inactivated. 

`I am not sure` means I have no idea how the exception handled by the someone. If the exception is suitable for this case in your thought, it is good to me too.

Reply:
I clearly make new exception for this case. c72621a